floss/api_hooks.py

```diff
@@ -307,6 +401,10 @@ def __call__(self, emu, api, argv):
 
 
 class MemsetHook:
+    """
+    Hook calls to memset: fill memory with a constant byte.
```

Comment:
thank you for these comments, they are helpful

Reply:
Thankyouu!!